backend/game_server/lib/helpers.ex

```diff
@@ -0,0 +1,2 @@
+defmodule BernacleServer.Helpers do
+end
```

Comment:
I propose 
#new_line - for missing new_line
#commas - for bad structure comas/spaces